### ISSUE TITLE
SIGRTMIN is not supported in Mac OS X and FreeBSD

### ISSUE THREAD

**Issue by kalloc** [samuelcolvin/arq#8]
```
y@wuza $ ./venv/bin/arq
Traceback (most recent call last):
  File "./venv/bin/arq", line 7, in <module>
    from arq.cli import cli
  File "/…/venv/lib/python3.5/site-packages/arq/__init__.py", line 6, in <module>
    from .worker import *  # type: ignore
  File "/…/venv/lib/python3.5/site-packages/arq/worker.py", line 39, in <module>
    SIG_SUPERVISOR = signal.SIGRTMIN + 7
AttributeError: module 'signal' has no attribute 'SIGRTMIN'

```

**Comment by samuelcolvin** [comment]
I don't have a mac so not that easy for me to test. Can you try with 

```
SIG_SUPERVISOR = signal.SIGRTMIN + 7
changed to
SIG_SUPERVISOR = signal.SIGUSR1
```

and let me know if that works?

**Comment by samuelcolvin** [comment]
I've created a branch with that change, so you can install with just

```
pip install -e git+git@github.com:samuelcolvin/arq.git@use-sigusr1#egg=arq
```

**Comment by kalloc** [comment]
@samuelcolvin thanks, It's working for me

**Comment by kalloc** [comment]
@samuelcolvin  Will you close the ticket?

**Comment by samuelcolvin** [comment]
fixed via #10 and deployed to pypi.
